draft-packages/badge/KaizenDraft/Badge/styles.module.scss

```diff
@@ -18,6 +21,7 @@ $small: $ca-grid;
   padding: 1px $kz-spacing-xs;
   min-width: 8px;
   text-align: center;
+  z-index: 2;
```

Comment:
Just to spread awareness in case, we have [a layers stylesheet](https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/styles/layers.scss) for z-index values and try to enforce this to avoid things getting messy over the long term with competing components etc.

But considering this looks like it won't be sharing a stacking context with the other components that use these values, this might be OK.

Reply:
Actually I will remove this addition, I previously animated using pseudo elements and this is no longer needed as I decided to animate the badge directly.

Reply:
I imagine the layers stylesheet are for more global values when incorporating many fixed elements. I wouldn't want an isolated stacking context to live there.